src/hit/api/evaluator.cpp

```diff
@@ -371,6 +377,14 @@ namespace hit {
         return ct.scale();
     }
 
+    CKKSCiphertext CKKSEvaluator::bootstrap(const CKKSCiphertext &ct, bool rescale_for_bootstrapping) {
+        VLOG(VLOG_EVAL) << "Bootstrapping ciphertext";
+        CKKSCiphertext ct_prime = bootstrap_internal(ct, rescale_for_bootstrapping);
+        ct_prime.bootstrapped_ = true;
+        print_stats(ct_prime);
+        return ct_prime;
+    }
```

Comment:
What is the relationship between `bootstrap` and `rescale_for_bootstrapping `?

Reply:
I've added docs to explain this in evaluator.h.